### PR TITLE
Add flag to fix relativeUserPkgDir

### PR DESCRIPTION
**Pull request by bnbarak**
# What
Per https://github.com/typicode/husky/issues/677, and @eduard-malakhov great explanation https://github.com/typicode/husky/issues/677#issuecomment-602520663

Fix a bug when running `yarn|npm install` from a sub `node_modules` in a monorepo/yarn workspaces project.

**Current:** the relative path to the project's root is set no `packages/someOther` when running from a package like `packages/someOther`.
**Expected**: To be able to force the relative path to the be the root of the project even when the `yarn|npm install` is running not from the root.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -328,6 +328,10 @@ npm install husky --save-dev
 
 Verify that your version of Git is `>=2.13.0`.
 
+#### Lerna Workspaces and Monorepos
```
Comment (bryantee) [inline README.md:331]: Seems like this fix would also address this happening in a Yarn Workspaces monorepo. Maybe we should call that out?

Comment (bnbarak) [reply README.md]: Changed to `Lerna, Monorepos, and Yarn Worksapces` to cover everything. What do you think?

Comment (bnbarak) [reply README.md]: Thanks :) Already updated the PR, what is your merge process?

Comment (bryantee) [reply README.md]: Maybe @typicode can chime in for merge process.

I'd love to test this out @bnbarak, what's the easiest way?

Comment (bnbarak) [reply README.md]: #### First lets recreate the issue on master:
0. close this example of a basic Lerna monorepo https://github.com/reggi/lerna-tutorial
1.  Setup Husky
2. Run `yarn` inside `packages/usage`
3. Open `.git/hooks/huskey.local.sh`, expect to see `cd "packages/usage"` // <-- **this is the problem**

#### Test the fix
4. Checkout this PR in the example project
5. Set the env variable `HUSKY_RUN_FROM_ROOT="true"` or `HUSKY_RUN_FROM_ROOT="1"`
6. Remove `.git/hooks`
7. Run `yarn` inside `packages/usage`
8. Open `.git/hooks/huskey.local.sh`, expect to see `cd "."` // <-- **this is the fix**

Comment (bnbarak) [reply README.md]: Piniging @lessen-inc and @typicode for their attention :)

**Comment by Faolain** [comment]
hey just bumping this, any reason why it hasn't been merged? @typicode @lessen-inc

**Comment by Faolain** [comment]
@typicode is there a reason this was closed and not merged? cc: @bnbarak @bryantee

**Pull request closed without merging**